src/runtime/runtime.cs

```diff
@@ -154,11 +148,6 @@ public class Runtime
         /// </summary>
         public static MachineType Machine { get; private set; }/* set in Initialize using python's platform.machine */
 
-        /// <summary>
-        /// Gets the machine architecture as reported by python's platform.machine().
-        /// </summary>
-        public static string MachineName { get; private set; }
-
         internal static bool IsPython2 = pyversionnumber < 30;
         internal static bool IsPython3 = pyversionnumber >= 30;
```

Comment:
If you are removing public members of Runtime, this makes this change breaking.

Reply:
Meh, I'll include them (with an `Obsolete` flag and just as a stringified version of the enum, absolutely not worth the effort exposing the actual `platform` values).

Reply:
@filmor should be fine as long as the values are compatible.

Reply:
They are not. But the values returned from Python in that regard are also not very pretty, just have a look at our dictionaries used to map the to something sensible. I highly doubt it's worth the effort.